Error = function(v0)
    print("ERROR:", v0);
    (nil):throw();
end;
return Error;
